cartridge-cli.lua

```diff
@@ -723,16 +733,15 @@ local function detect_version(source_dir)
         return nil
```

Comment:
I sure that this function should be renamed to `detect_git_version`, it detects version only from git.

Reply:
Done